governance.md

```diff
@@ -12,7 +12,7 @@ photos:
 {% assign people = site.data.people %}
 
 # Governance Committee
-The OLS Governance Committee is a dedicated group of individuals appointed to provide strategic oversight and guidance in shaping the governance framework of OLS. This committee provides advisory support in decision-making, transparency, and sustainability within OLS.
+The OLS Governance Committee is a dedicated group of individuals appointed to provide strategic oversight and guidance in shaping the governance framework of OLS. This committee provides advisory support in decision-making, transparency, and sustainability within OLS. You can also [read the OLS governance guiding documents](https://github.com/open-life-science/ols-governance)
```

Comment:
```suggestion
The OLS Governance Committee is a dedicated group of individuals appointed to provide strategic oversight and guidance in shaping the governance framework of OLS. This committee provides advisory support in decision-making, transparency, and sustainability within OLS. You can also [read the OLS governance guiding documents]({{ site.github.owner_url }}/ols-governance)
```

Reply:
thanks!! 🌹  I wondered if I was missing a nifty variable like that

Reply:
yeah. Old habits die hard. Variablesss!